### ISSUE TITLE
VoiceOver support for Themes cards

### ISSUE THREAD

**Issue by ctarda** [wordpress-mobile/WordPress-iOS#8697]
### Expected behavior
Cards in Themes should provide better integration with VoiceOver.

![img_2548](https://user-images.githubusercontent.com/2722505/36570314-4b468f38-17e6-11e8-8f4a-871ee601c2ad.PNG)

1.- The whole area marked as (1) should be exposed as a single element, and read "Current Theme: THEME_NAME"
2.- Individual themes should be exposed to voice over (2), and provide a description of the theme (if available)
3.- The Ellipsis icon (3) should be exposed to voice over.

### Actual behavior
When tapping on a theme, the whole screen is selected 

### Steps to reproduce the behavior
Navigate to Themes, activate VoiceOver, tap on any theme

##### Tested on [device], iOS [version], WPiOS [version]
